### PR TITLE
Fix install

### PR DESCRIPTION
**Pull request by calispac**
The owner of CTS package has changed. the path is corrected.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+build/
+__pycache__
```
Comment (dneise) [inline .gitignore]: isn't this a duplicate of line 5?

File: .gitignore [hunk 2]
```diff
@@ -103,3 +105,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# Baobab slurm script
+job.sh
```
Comment (dneise) [inline .gitignore]: pep8: a file should end with a newline

**Comment by dneise** [comment]
Ah what is `job.sh` and why do we gitignore it?

File: .gitignore (hunk 2 above)
Comment (calispac) [reply .gitignore]: it was a file I am using on a cluster machine.

**Comment by calispac** [comment]
a file that was used for a cluster machine

**Comment by dneise** [comment]
Looks pretty fine, I'd say we can merge as soon as travis is done

**Pull request merged**